atomic_reactor/plugins/post_group_manifests.py

```diff
@@ -208,6 +209,40 @@ def build_list(self, manifests):
                 ],
         }, indent=4)
 
+    def check_existing_vr_tag(self):
+        """
+        Checks if version-release tag (primary not floating tag) exists already,
+        and fails plugin if it does.
+        """
+        primary_images = get_primary_images(self.workflow)
+
+        if not primary_images:
+            return
+
+        vr_image = None
+        for image in primary_images:
+            if '-' in image.tag:
+                vr_image = image
+                break
+        if not vr_image:
```

Comment:
could be `else:`, as in `for ... else ...`, and then you also wouldn't need to initialise it to `None` first. (But not everyone likes that construct...)